lib/pliny/tasks/db.rake

```diff
@@ -34,7 +34,7 @@ begin
     task :seed do
       if File.exist?('./db/seeds.rb')
         database_urls.each do |database_url|
-          Sequel.connect(database_url)
+          DB = Sequel.connect(database_url)
```

Comment:
this is iterating over `database_urls`, which means it'll keep setting the same constant multiple times when more than 1 urls are present. Is that intended?